src/router/index.js

```diff
@@ -1,36 +1,21 @@
-import Vue from 'vue';
-import VueRouter from 'vue-router';
-
+import {
+    createRouter, createMemoryHistory, createWebHistory, createWebHashHistory } from 'vue-router'
 import routes from './routes';
-import ga from './analytics';
-
-Vue.use(VueRouter);
-
-const Router = new VueRouter({
-    /*
-     * NOTE! Change Vue Router mode from quasar.conf.js -> build -> vueRouterMode
-     *
-     * When going with "history" mode, please also make sure "build.publicPath"
-     * is set to something other than an empty string.
-     * Example: '/' instead of ''
-     */
-
-    // Leave as is and change from quasar.conf.js instead!
-    mode: process.env.VUE_ROUTER_MODE,
-    base: process.env.VUE_ROUTER_BASE,
-    scrollBehavior: () => ({ y: 0 }),
-    routes
-});
 
-Router.afterEach(to => {
-    Vue.gtm.trackView(
-        to.path,
-        to.name,
-        {
-            cid: ga.createSessionId(),
-            path: to.path
-        }
-    );
-});
+export default function (/* { store, ssrContext } */) {
+    const createHistory = process.env.SERVER
+      ? createMemoryHistory
+      : process.env.VUE_ROUTER_MODE === 'history' ? createWebHistory : createWebHashHistory
```

Comment:
I think we should always use history mode. so this will `createWebHistory` only

Reply:
thanks. updated